build/source/driver/multi_driver.f90

```diff
@@ -440,89 +615,90 @@ real(dp),parameter        :: safety = 0.01_dp               ! safety factor to e
  ! ****************************************************************************
  do iHRU=1,nHRU
 
-  ! print progress
-  !print*, 'iHRU = ', iHRU
-
-  ! assign pointers to HRUs
-  time_data => time_hru(iHRU)
-  forc_data => forc_hru(iHRU)
-  attr_data => attr_hru(iHRU)
-  type_data => type_hru(iHRU)
-  mpar_data => mpar_hru(iHRU)
-  mvar_data => mvar_hru(iHRU)
-  indx_data => indx_hru(iHRU)
-
   ! identify the area covered by the current HRU
```

Comment:
Suggested clarification:
`! identify the fraction of the total basin area covered by the current HRU`